File: C/Practice/PALINDROME.c
#include<stdio.h>
int main()
{
int a,s=0,d;
printf("enter the number\n");
scanf("%d",&a);
int t=a;
while(a!=0)
{
	d=a%10;
	s=(s*10)+d;
	a=a/10;
}
printf("%d",s);
if(t==s)
{
printf("Palindrome");
}
else{
printf("not a palindrome");
}
return 0;
}

File: Java/Java_Programs/PALINDROME.java
import java.io.*;
class PALINDROME
{
    public static void main(String args[])throws IOException
    {
        InputStreamReader reader=new InputStreamReader(System.in);
        BufferedReader input=new BufferedReader(reader);
        System.out.println("Enter the number ");
        int n=Integer.parseInt(input.readLine());
        int t=n;
        int s=0,d;
        while(n!=0)
        {
            d=n%10;
            s=(s*10)+d;
            n=n/10;
        }
        if(s==t)
        {
            System.out.println(t+" is a palindrome");
        }
        else
        {
            System.out.println(t+" is not a palindrome");
        }
    }
}